pkg/cmd/render/bootstrap_ip_linux.go

```diff
@@ -0,0 +1,67 @@
+package render
```

Comment:
Probably not, but I was trying to be a good idiomatic Go citizen and ensure that the codebase will still compile on non-linux platforms (e.g. darwin). Without architecture build tags for this the operator itself will no longer compile on macos because of the native netlink dependency.